src/components/YieldFarmChainError.tsx

```diff
@@ -1,24 +1,33 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Close from 'assets/chat/group-chat/close.svg?react';
 import { ButtonV2, ItemHV2, ItemVV2 } from './reusables/SharedStylingV2';
-import styled from 'styled-components';
+import styled, { useTheme } from 'styled-components';
 import { ethers } from 'ethers';
 import { appConfig } from 'config/index.js';
 import { useAccount } from 'hooks';
 
-const YieldFarmChainError = () => {
+const YieldFarmChainError = ({ onClose }) => {
   const { account, chainId: currentChainId, switchChain } = useAccount();
 
+  const theme = useTheme();
+
   const handleChainChange = () => {
     const chainIdToPass = appConfig.allowedNetworks[0];
 
     if (currentChainId !== 1 && currentChainId !== 11155111) {
-      console.debug('Current Chain ID ', currentChainId);
-      console.debug('Chain Id to pass', chainIdToPass);
+      console.info('Current Chain ID ', currentChainId);
+      console.info('Chain Id to pass', chainIdToPass);
       switchChain(appConfig.coreContractChain);
     }
   };
 
+  useEffect(() => {
+    if (currentChainId === 1 || currentChainId === 11155111) {
```

Comment:
Why we are these specific values like 1 and 11155111? 

What does these justify here? 

Are we completely sure these will remain fixed? If yes then please store these values in a const with proper comments letting the developer know more about it.



```
// contants.ts

// The meaning and purpose of these id are...

const ChainIdTypeA: ChainIdType = 1;

const chainIdTypeB: ChainIdType = 11155111;

// types.ts

export type ChainIdType = 1 | 11155111

```
P.S. You could use more meaningful names.

Reply:
Fixed in new commit